system-tests/services/test-apps.js

```diff
@@ -1677,5 +1677,186 @@ describe("Services", function() {
+      cy.get('input[name="portDefinitions.1.name"]').type("mapped");
+
+      cy
+        .get('input[name="portDefinitions.1.portMapping"]')
+        .parents(".form-control-toggle")
+        .click();
+
+      cy
+        .get('input[name="portDefinitions.1.automaticPort"]')
+        .parents(".form-control-toggle")
+        .click();
+
+      cy.root().getFormGroupInputFor("Host Port").type("4200");
+
+      cy.get("label").contains("JSON Editor").click();
+
+      cy.get("#brace-editor").contents().asJson().should("deep.equal", [
+        {
+          id: `/${Cypress.env("TEST_UUID")}/${serviceName}`,
+          cmd: command,
+          cpus: 0.5,
+          mem: 32,
+          instances: 1,
+          container: {
+            type: "DOCKER",
+            docker: {
+              image: containerImage,
+              network: "USER",
+              portMappings: [
+                {
+                  name: "http",
+                  containerPort: 8080
+                },
+                {
+                  name: "mapped",
+                  hostPort: 4200,
+                  containerPort: 8080,
+                  protocol: "tcp"
+                }
+              ]
+            }
+          },
+          ipAddress: {
+            networkName: "dcos"
+          },
+          requirePorts: true
+        }
+      ]);
+
+      cy.get("button").contains("Review & Run").click();
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Service ID")
+        .contains(`/${Cypress.env("TEST_UUID")}/${serviceName}`);
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Container Runtime")
+        .contains("Docker Engine");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Instances")
+        .contains(1);
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("CPU")
+        .contains("0.5");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Memory")
+        .contains("32 MiB");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Container Image")
+        .contains(containerImage);
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Command")
+        .contains(command);
+
+      cy
+        .root()
+        .configurationSection("Network")
+        .configurationMapValue("Network Type")
+        .contains("USER");
+
+      cy
+        .root()
+        .configurationSection("Network")
+        .configurationMapValue("Network Name")
+        .contains("dcos");
+
+      cy
+        .root()
+        .configurationSection("Service Endpoints")
+        .then(function($section) {
+          expect($section.get().length).to.equal(1);
+          const numOfCols = 6;
```

Comment:
In general I wanted to avoid keeping such a generic code inside the tests. For this kind of uses, I introduced the `.getTableColumn()` function (check `_support/index.js`). 

You can use it like so:

```js
$section.find("table").getTableColumn(0).should("deep.equal", [
  "http",
  "Not configured",
  ...
]);
```

And if that function is not working for you, I would recommend to create a new utility function that could (for example) extract all cells and rows of the table.